### ISSUE TITLE
jpeg options

### ISSUE THREAD

**Issue by coolbeanz10101010** [Iksas/media-import-2#2]
Is there not a way to upload JPEG? my converter wont let me convert pdf to JPG

**Comment by Iksas** [comment]
Yes, the plugin should be able to import both `.JPG` and `.JPEG` files. 

If you experience a specific problem while trying to import `.JPG` or `.JPEG` files, please let me know, so I can fix it.

By the way, `.JPG` files are exactly the same as `.JPEG` files, just with a different file name. You can just rename the files to "convert" between `.JPG` and `.JPEG`.

**Comment by coolbeanz10101010** [comment]
So i added the addon. I want media for both front and back of card. Do I save all images in a single folder and select that folder? Will it convert each jpeg into a card? What about the backside media? Thanks in advance

**Comment by Iksas** [comment]
It's currently not possible to add different media to the front and back of a card. You can only add the same media to the front and back of a card. To do that, save all images in a single folder, and select the folder.

I'm interested to implement this feature at some point in the future.

**Comment by Iksas** [comment]
A feature to add multiple media files to a single card has been implemented in version 3.2 of the addon. 

To use it, all images have to be saved in the same folder, and the file name of the second media file must end in `_2` (for example `image.jpg` and `image_2.jpg`).

Inside the addon, the first media file can be selected with the `Media` option, as usual. The second media file can be selected with the new `Media_2` option.
